app/Http/routes.php

```diff
@@ -56,3 +56,5 @@
     // signups
     Route::post('signups', 'Api\SignupsController@store');
 });
+
+Route::get('api/v2/activity', 'Api\ActivityController@index');
```

Comment:
@sbsmith86 I put it outside so a user doesn't have to be authorized to hit this endpoint